pkg/controller/migmigration/restore.go

```diff
@@ -271,7 +279,7 @@ func (t *Task) deleteMigrated() error {
 	}
 
 	listOptions := k8sclient.MatchingLabels(map[string]string{
-		MigratedByLabel: string(t.Owner.UID),
+		MigPlanLabel: string(t.PlanResources.MigPlan.UID),
```

Comment:
+1 on labeling restored resources with the MigPlan UID so that we can remove everything that's been migrated so far.